Libraries/Geolocation/RCTLocationObserver.m

```diff
@@ -144,6 +144,14 @@ - (void)beginLocationUpdatesWithDesiredAccuracy:(CLLocationAccuracy)desiredAccur
   if ([[NSBundle mainBundle] objectForInfoDictionaryKey:@"NSLocationAlwaysUsageDescription"] &&
     [_locationManager respondsToSelector:@selector(requestAlwaysAuthorization)]) {
     [_locationManager requestAlwaysAuthorization];
+
+    // On iOS 9+ we also need to enable background updates
+    NSArray* backgroundModes  = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"UIBackgroundModes"];
+    if(backgroundModes && [backgroundModes containsObject:@"location"]) {
+      if([_locationManager respondsToSelector:@selector(setAllowsBackgroundLocationUpdates:)]) {
+        [_locationManager setAllowsBackgroundLocationUpdates:YES];
+      }
+    }
```

Comment:
There is also a case to be made to expose this property to JS, as there are times when you might want to toggle the background updates dynamically, but it didn't seem to fit in with the Mozilla geolocation API, so I thought this fit best with what is there already.